apps/HedgeDoc.md

```diff
@@ -2,4 +2,129 @@
 
 Was CodiMD: create real-time collaborative markdown notes
 
-[GitHub](https://github.com/hedgedoc/hedgedoc)
\ No newline at end of file
+
+[GitHub](https://github.com/hedgedoc/hedgedoc)
+
+# Migration
+
+## Start PostgreSQL:
+```
+
+docker run -d --name pg \
+ --hostname pg -e POSTGRES_DB=db -e POSTGRES_USER=user -e POSTGRES_PASSWORD=password \
+ --health-cmd="PGPASSWORD=password pg_isready -h pg -p 5432 -U user -d db" --health-interval=3s --health-timeout=3s \
+ postgres:12
+
+```
+
+## Start the application:
+```
+
+docker run -d --name app --link pg:pg \
+ -p 3000:3000 \
+ -e CMD_DB_URL=postgres://user:password@pg:5432/db \
+ -e CMD_ALLOW_FREEURL=true -e CMD_ALLOW_EMAIL_REGISTER=true -e CMD_DEFAULT_PERMISSION=editable \
+ -e CMD_URL_ADDPORT=false -e CMD_PROTOCOL_USESSL=true -e CMD_SESSION_SECRET=Postgres2YugabyteDB   \
+ linuxserver/hedgedoc
+
+```
+
+## Test the application:
+
+Create some docs using the API
+```
+
+time for f in $( find /usr -name README )
+do
+curl -X POST "http://localhost:3000/new" -H "Content-Type: text/markdown" -d "@$f"
+done
+
+curl -X POST "http://localhost:3000/new/MyNote" -H "Content-Type: text/markdown" -d "My node written at $(date)"
+curl -X GET "http://localhost:3000/MyNote/info"
+
+```
+
+## Stop the application
+(remove the container - if there were some persitent files, put them in external volume):
+```
+
+docker stop app
+docker rm   app
+
+```
+
+## Start YugabyteDB
+
+```
+
+docker run -d --name yb  \
+ --hostname yb -e YSQL_DB=db -e YSQL_USER=user -e YSQL_PASSWORD=password \
+ --health-cmd="PGPASSWORD=password postgres/bin/pg_isready -h yb -p 5433 -U user -d db" --health-interval=3s --health-timeout=3s \
+ yugabytedb/yugabyte:2024.1.1.0-b137 \
+ bash -c 'rm -rf /tmp/.yb.* ; yugabyted start --enable_pg_parity_tech_preview --background=false --tserver_flags=ysql_colocate_database_by_default=true'
+
+```
+
+## Migrate with YugabyteDB Voyager
+Start a container with the YugabyteDB voyager image and run the migration steps
+
+```
+
+docker run -it --rm --name ybv --link pg:pg --link yb:yb \
+ yugabytedb/yb-voyager:1.7.2 \
+ bash -c '
```

Comment:
Hey @FranckPachot ! , for docker installations of voyager, we also provide a wrapper script [yb-voyager-docker](https://docs.yugabyte.com/preview/yugabyte-voyager/install-yb-voyager/#:~:text=Download%20the%20script,run%20Voyager%20commands.) that essentially allows you to use it just like the non-dockerised version. It mounts the export-dir and passes the flags to the container appropriately. 

Did you happen to try that and face any difficulties?